README.md

```diff
@@ -74,6 +75,16 @@ Bigcommerce::configure(array(
 ));
 ```
 
+oAuth
```

Comment:
typically spelled OAuth
I'd also make this and Basic Auth at least a subheading, formatting-wise, for easier skimming